modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -1,6 +1,7 @@
 // This is included in the following assemblies:
 //
-// networking/routes/route-configuration.adoc
+// * networking/routes/route-configuration.adoc
+// * microshift_networking/microshift-configuring-routes.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="creating-re-encrypt-route-with-custom-certificate_{context}"]
```

Comment:
same question about ID matching filename

Reply:
same here, not a popular xref or include anywhere; updated